src/test/resources/features/actions/line_mover.feature

```diff
@@ -1,44 +1,59 @@
 Feature: Line mover
-  Scenario Outline: Move rules up
+  Scenario Outline: Permutation up/down between rules sections
     Given a snakemake project
     Given I open a file "foo1.smk" with text
     """
-    <rule_like1> NAME1:
-        <rule_section1>: <content>
-    <rule_like2> NAME2:
-        <rule_section2>:
-            <content>
+    <rule_like2> <name2>:
+        <rule_section2>: <content2>
+    <rule_like1> <name1>:
+        <rule_section1>:
+            <content1>
     """
-    When I put the caret at <rule_like2> NAME2:
+    When I put the caret at <rule_like1> <name1>:
     And I invoke MoveStatementUp action
     Then editor content will be
     """
-    <rule_like2> NAME2:
-        <rule_section2>:
-            <content>
-    <rule_like1> NAME1:
-        <rule_section1>: <content>
+    <rule_like1> <name1>:
+        <rule_section1>:
+            <content1>
+    <rule_like2> <name2>:
+        <rule_section2>: <content2>
+    """
+    When I put the caret at <rule_like1> <name1>:
+    And I invoke MoveStatementDown action
+    Then editor content will be
+    """
+    <rule_like2> <name2>:
+        <rule_section2>: <content2>
+    <rule_like1> <name1>:
+        <rule_section1>:
+            <content1>
     """
     Examples:
-      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content         |
-      | rule        | rule        | input          | output         | "file.txt"      |
-      | checkpoint  | rule        | input          | output         | "file.txt"      |
-      | subworkflow | rule        | workdir        | output         | "file.txt"      |
-      | rule        | checkpoint  | input          | output         | "file.txt"      |
-      | checkpoint  | checkpoint  | input          | output         | "file.txt"      |
-      | subworkflow | checkpoint  | workdir        | output         | "file.txt"      |
-      | rule        | subworkflow | input          | workdir        | "file.txt"      |
-      | checkpoint  | subworkflow | input          | workdir        | "file.txt"      |
-      | subworkflow | subworkflow | workdir        | workdir        | "file.txt"      |
-
-  Scenario Outline: Permutation up between rule and comment sections
+      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content1    | name1 | name2   | content2   |
```

Comment:
`content1` value is always `file.txt` here, `name1` is always `NAME1`. It is a but in test or redundant variables?

Reply:
Also you could use values like `input: "foo"` or `input:\n    "foo"` to make section with content single/multi line

Reply:
'I put the caret at (.+)'  doesn't work correctly with line with \n